### PR TITLE
templates: welcome page

### PR DESCRIPTION
**Pull request by atrisovic**
Signed-off-by: Ana Trisovic <ana.trisovic@gmail.com>

### CODE REVIEW

File: cernopendata/modules/fixtures/data/articles/welcome/welcome.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+FIXME PRELIMINARY TEXT
+
+With the growing demand of open science data and increased interest in the CERN Open Data portal, we were working hard on modernizing our web presence. Here, we are telling you more about the changes we made and how you can use the updated portal.
+
+ <img src="../../../../static/assets/header-image.svg" width="70%" align="middle">
```
Comment (diegodelemos) [inline cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: Does this render for you?

screen shot 2017-11-17 at 10 44 26 am

Most probably it is related with https://stackoverflow.com/questions/32123484/svg-use-external-source-not-working-on-firefox/32124010#32124010 (discovered by @hjhsalo) you could either try that fix or convert the `svg` to `png`

Comment (ioannistsanaktsidis) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: Also the dots should be removed and the link should be ```/static/.../```.

Comment (atrisovic) [reply cernopendata/modules/fixtures/data/articles/welcome/welcome.md]: now fixed :+1:

**Comment by ioannistsanaktsidis** [comment]
@atrisovic please add 
```
"screenshot": <path> E.g /static/etc...
"featured": 1
```
and rebase against latest master so as to merge it .

**Comment by ioannistsanaktsidis** [comment]
@ArtemisLav  @tiborsimko  is this one ok ? Should I go  on ?

**Pull request merged**